src/main/kotlin/com/aroundme/content/ContentDTO.kt

```diff
@@ -0,0 +1,11 @@
+package com.aroundme.content
+
+import java.time.LocalDateTime
+
+data class ReadContentDTO (
+    val id: Long? = null,
+    val category: String,
+    val content: String,
+    val media: String,
+    val addedAt: LocalDateTime
```

Comment:
createdTime?

Reply:
How about createdAt? 
On the other hand, I think createdTime is more appropriate because it also stores time units.